xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -956,112 +956,117 @@ Rail transport, Roads and Road transport
-				</xsd:restriction>
-			</xsd:complexContent>
-		</xsd:complexType>
-	</xsd:element>
-	<xsd:complexType name="RampEquipment_VersionStructure" abstract="false">
-		<xsd:annotation>
-			<xsd:documentation>Type for a RAMP EQUIPMENT.</xsd:documentation>
-		</xsd:annotation>
-		<xsd:complexContent>
-			<xsd:extension base="AccessEquipment_VersionStructure">
-				<xsd:sequence>
-					<xsd:group ref="RampEquipmentGroup"/>
-				</xsd:sequence>
-			</xsd:extension>
-		</xsd:complexContent>
-	</xsd:complexType>
-	<xsd:group name="RampEquipmentGroup">
-		<xsd:annotation>
-			<xsd:documentation>Elements for RAMP EQUIPMEN/</xsd:documentation>
-		</xsd:annotation>
-		<xsd:sequence>
-			<xsd:element name="Length" type="LengthType" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Length of ramp in metres.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="Gradient" type="xsd:positiveInteger" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Gradient of ramp in degrees.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="GradientType" type="GradientEnumeration" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Gradient of ramp fixed values.+v1.1</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="Pedestal" type="xsd:boolean" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Whether ramp is on a pedestal.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="HandrailHeight" type="LengthType" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Height of handrail.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="HandrailType" type="HandrailEnumeration" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Type of handrail.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="TactileGuidanceStrips" type="xsd:boolean" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Whether ramp has tactile guidance strips.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="VisualGuidanceBands" type="xsd:boolean" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Whether ramp has visual guidance bands or guidance strips.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="Temporary" type="xsd:boolean" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Whether ramp is temporary or permanent.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="SuitableForCycles" type="xsd:boolean" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Whether equipment is suitable for cycles.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-		</xsd:sequence>
-	</xsd:group>
+  <xsd:element name="RampEquipment" substitutionGroup="AccessEquipment">
```

Comment:
This is not good.

Reply:
what do you suggest? redo the change without the indentation problem?